.circleci/config.yml

```diff
@@ -128,11 +128,17 @@ commands:
           command: |
             FOUNDRY_REPO="foundry-rs/foundry"
             FOUNDRY_VERSION="<< parameters.version >>"
+            # Make authenticated requests when the Github token is available
+            if [[ -n "$GITHUB_ACCESS_TOKEN" ]]; then
+              # shellcheck disable=SC2089
+              HEADER="--header 'Authorization: Bearer ${GITHUB_ACCESS_TOKEN}'"
+            fi
+            # shellcheck disable=SC2086,SC2090
             FOUNDRY_RELEASE_SHA=$(curl \
               --silent \
               --fail \
               --show-error \
-              --header "Authorization: Bearer ${GITHUB_ACCESS_TOKEN}" \
+              ${HEADER} \
```

Comment:
This is intentionally unquoted so it will be `null` if not defined. Use `"${HEADER}" here would cause it to be evaluated to `""` which would in turn cause an error on the `curl` command.